flink-libraries/flink-table/src/main/scala/org/apache/flink/table/plan/nodes/datastream/DataStreamOverAggregate.scala

```diff
@@ -136,13 +229,13 @@ class DataStreamOverAggregate(
             namedAggregates,
             inputType)
 
-          inputDS
+        inputDS
           .keyBy(partitionKeys: _*)
           .process(processFunction)
           .returns(rowTypeInfo)
           .name(aggOpName)
           .asInstanceOf[DataStream[Row]]
-        }
+      } // global non-partitioned aggregation
```

Comment:
I agree...but that was from the code that already existed...from the previous merges.